src/pymovements/stimulus/stimulus.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2022-2024 The pymovements Project Authors
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+"""Provides the StimulusDataFrame class."""
+from __future__ import annotations
+
+
+class Stimulus:
+    """Stimulus base class.
+
+    Initialize the Stimulus object.
+
+    path : str | Path | DatasetPaths
+        Path to the dataset directory. You can set up a custom directory structure by passing a
+        :py:class:`~pymovements.DatasetPaths` instance.
+    """
+
+    def __init__(
+            self,
+    ) -> None:
+        self.stimulus: list[Stimulus] = []
```

Comment:
I'm not sure if I follow. Why would a stimulus be a list of stimuli?
I think at this stage it's hard to know what this class should actually do, as we only have one type of stimulus and don't know yet what common attributes and functionalities would be needed across stimulus types.

I would just remove the init for now and create an empty class.

Reply:
we could also remove the class completely for now, just as you like

Reply:
the idea was similar to the gazedataframe but I deleted it for now